packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDTutorial/PhotoCaptureHUDTutorial.tsx

```diff
@@ -0,0 +1,129 @@
+import { CSSProperties } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, DynamicSVG } from '@monkvision/common-ui-web';
+import { CaptureAppConfig } from '@monkvision/types';
+import { styles } from './PhotoCaptureHUDTutorial.styles';
+import { TutorialViews } from '../../hooks';
+import { usePhotoCaptureHUDButtonBackground } from '../hooks';
+import { SightGuideline } from '../PhotoCaptureHUDElementsSight';
+import { arrowGuidelineSVG, arrowSightTutorialSVG } from '../../../assets';
+
+const TRANSLATION_PREFIX = 'photo.hud.tutorial';
```

Comment:
This is not a good idea to factorize translation key labels like that. The reasons behind it are simple :
- It isn't _that_ complex to always add the complete translation key every time, so it's fine to do it manually every time
- Also, when trying to clean up the unused translation labels in the project, usually what the developers will do is CTRL + F in the whole project and look the usages of `photo.hud.tutorial.next` to see if the translation label is used or not. If you factorize the prefix like that you won't get the result.

So to conclude : you can leave the whole translation key every time, it's fine